src/lib/make-toolbox-xml.js

```diff
@@ -1,17 +1,18 @@
 import ScratchBlocks from 'scratch-blocks';
+import {defaultColors} from './themes';
 
 const categorySeparator = '<sep gap="36"/>';
 
 const blockSeparator = '<sep gap="36"/>'; // At default scale, about 28px
 
 /* eslint-disable no-unused-vars */
-const motion = function (isInitialSetup, isStage, targetId) {
+const motion = function (isInitialSetup, isStage, targetId, colors) {
     const stageSelected = ScratchBlocks.ScratchMsgs.translate(
         'MOTION_STAGE_SELECTED',
         'Stage selected: no motion blocks'
     );
     return `
-    <category name="%{BKY_CATEGORY_MOTION}" id="motion" colour="#4C97FF" secondaryColour="#3373CC">
+    <category name="%{BKY_CATEGORY_MOTION}" id="motion" colour="${colors.primary}" secondaryColour="${colors.tertiary}">
```

Comment:
Is `colors.tertiary` always used to fill in a "secondary" color?

Reply:
For the category styling, yes. 

These aren't all of the uses, but for the block colors, `scratch-blocks` uses:
- primary for the block background color
- secondary for the dropdown background color
- tertiary for the border color

These categories only have two colors for the circle icon: background (primary) and border (secondary). It is a bit unfortunate that the block's tertiary matches up to the category's secondary. Any suggestions to make this less confusing?

Reply:
Gotcha, that makes sense. I probably knew that at some point in the past...

I think it could make sense to add a comment about this if you see a good place for it, but once it's established it'll go from "hmm, I wonder if this is a mistake" to "hmm, it's a bit odd that it worked out this way" -- less of a concern :)

Reply:
👍 Added some comments to the code.